prov/verbs/src/fi_verbs.h

```diff
@@ -756,6 +756,8 @@ int vrb_ep_destroy_xrc_qp(struct vrb_xrc_ep *ep);
 
 int vrb_xrc_close_srq(struct vrb_srq_ep *srq_ep);
 int vrb_sockaddr_len(struct sockaddr *addr);
+void vrb_set_sid(enum rdma_port_space ps,
+        uint16_t port, struct sockaddr_ib *sib);
```

Comment:
github needs a way to comment directly on the commit message...  The commit message needs a better description.  I don't think the changes are related to ipoib.  They seem to just generate native IB addresses for all <port git, pkey> pairs in the system, independent of whether ipoib is enabled on that pair or not.  Obviously, the reason for the change is to handle the case where ipoib is not installed, and I would capture that as the motive.

Reply:
Thanks. I have updated the commit message as you suggested.